UI/src/view/navigationpanel/searchBox.js

```diff
@@ -83,7 +83,7 @@
         });
         eventbus.trigger('coordinates:selected', { lon: result.lon, lat: result.lat });
       } else if (result.resultType.indexOf("Mtstop")>-1) {
-        window.location.hash="#massTransitStop/"+result.nationalId;
+        window.location.hash="#massTransitStop/"+result.id;
```

Comment:
There is another option to maintain the old behavior by simply using:
```javascript
window.location.hash="#massTransitStopNationalId/"+result.nationalId;
```

But for me what is already done is good too

Reply:
attention to this, the search for a bus stop is always using the externalID/nationalID.
When is mentioned Bus Stop ID e the national ID and not the id of the asset.
That result.id is what value?

Reply:
Reverted to used NationalId